test/unit/rules/no-bare-strings-test.js

```diff
@@ -100,6 +87,18 @@ generateRuleTests({
       template: '&#8377;',
     },
 
+    {
+      // extend the allowlist list
+      config: ['/', '"'],
+      template: '{{t "foo"}} / &lpar;"{{name}}"&rpar;',
+    },
+
+    {
+      // extend the allowlist list
+      config: { allowlist: ['/', '"'] },
+      template: '{{t "foo"}} / &lpar;"{{name}}"&rpar;',
+    },
```

Comment:
Should add a valid test case that ignores the default allow list and uses something from it.